.github/workflows/continous-integration.yml

```diff
@@ -686,6 +686,92 @@ jobs:
         run: |
           make test-integration
 
+  sequential_integration_test:
+    name: Run Sequential Integration Tests
```

Comment:
Run Integration Tests Sequentially?

Reply:
Ah, it's called like this because the tests themselves are marked with a pytest marker called `sequential`, this is why I chose this name 😄